GwApiService/src/main/java/us/gameandwatching/gwapi/service/calc/EloCalculator.java

```diff
@@ -0,0 +1,81 @@
+package us.gameandwatching.gwapi.service.calc;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import us.gameandwatching.gwapi.core.EloPair;
+import us.gameandwatching.gwapi.core.MatchResult;
+
+@Singleton
+public class EloCalculator {
+
+  @Inject
+  public EloCalculator() {
+
+  }
+
+  public EloPair updateEloRating(EloPair eloPair) {
+    double i1 = eloPair.getPlayer1Rating();
```

Comment:
Yea, I agree with the more descriptive names.  I named them the same as the formula in the paper I was reading to make it easier to follow along, but i'll fix it up